### PR TITLE
[dcl.decl, temp.deduct.type] Fix inappropriate uses of \grammarterm.

### PR DESCRIPTION
**Pull request by Eelis**
No visible change.

### CODE REVIEW

File: source/declarators.tex [hunk 1]
```diff
@@ -2105,16 +2105,16 @@
 
 \pnum
 In the \grammarterm{function-body}, a
-\grammarterm{function-local predefined variable} denotes a block-scope object of static
+\techterm{function-local predefined variable} denotes a block-scope object of static
```
Comment (zygoloid) [inline source/declarators.tex]: Seems like `\defn` would be a better choice here.

File: source/declarators.tex [hunk 2]
```diff
@@ -2139,7 +2139,7 @@
 \grammarterm{function-body}
 is of the form
 \tcode{= default ;}
-is called an \grammarterm{explicitly-defaulted} definition.
+is called an \techterm{explicitly-defaulted} definition.
```
Comment (zygoloid) [inline source/declarators.tex]: Change this to `\defnx{explicitly-defaulted}{definition!function!explicitly-defaulted}`, and remove the `\indextext` above.

File: source/declarators.tex (hunk 1 above)
Comment (Eelis) [inline source/declarators.tex]: How would you feel about `\defnx{function-local predefined variable}{variable!function-local predefined}`?

Comment (zygoloid) [reply source/declarators.tex]: Sounds good to me.

**Comment by Eelis** [comment]
Rebased and amended.

**Pull request merged**